.github/linters/.markdown-lint.yml

```diff
@@ -0,0 +1,366 @@
+---
```

Comment:
I would add this config once we enable MARKDOWN linter (as part of that specific task), not now. Change is irrelevant for this PR? Since .md files are not checked?

Reply:
Removed from this PR